traffic_ops/client/cachegroup_parameters.go

```diff
@@ -67,6 +67,28 @@ func (to *Session) getCacheGroupParameters(route, queryParams string) ([]tc.Cach
 	return data.Response, reqInf, nil
 }
 
+// GetAllCacheGroupParameters Gets all Cachegroup Parameter associations
+func (to *Session) GetAllCacheGroupParameters() ([]tc.CacheGroupParametersResponseNullable, ReqInf, error) {
+	route := fmt.Sprintf("%s/", API_v1_CacheGroupParameters)
+	return to.getAllCacheGroupParameters(route, "")
+}
+
+func (to *Session) getAllCacheGroupParameters(route, queryParams string) ([]tc.CacheGroupParametersResponseNullable, ReqInf, error) {
```

Comment:
just curious why the need for the two methods?

Reply:
oh originally i was going to mirror the get methods above but realized i only needed the one but left it split even though the second one isnt reused. ill combine them again since theres no need for 2